src/Fetch.mli

```diff
@@ -88,6 +90,14 @@ type requestRedirect =
   | Error
   | Manual
 
+module AbortController : sig
```

Comment:
There should be a doc comment warning that this is an experimental API on the module and on each function.